code/dynamic_programming/src/fibonacci/fibonacci.cpp

```diff
@@ -0,0 +1,42 @@
+/* Part of Cosmos by OpenGenus Foundation */
+
+#include <string>
+#include <sstream>
+#include <iostream>
+
+using namespace std;
```

Comment:
Don't use this.

Reply:
Do you suggest to put the namespace inside the main function?